app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -182,6 +159,22 @@ public void onClick(View v) {
     }
 
     /**
+     * Set response when clicking in the bottom navigation
+     */
+    public void setBottomNavigationOnClickListeners(View v) {
```

Comment:
This doesn't really define what the function does. This toggles the Visualization Fragment so it should be named `toggleVisualizationFragment(..)` or `toggleVisualizationFragmentOnClick(..)`